runtime/opt/taupage/init.d/06-update-sysctl.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import logging
+import sys
+import subprocess
+
+from taupage import configure_logging, get_config
+
+SYSCTL_WHITELIST = ['vm.dirty_background_ratio', 'vm.dirty_ratio', 'vm.overcommit_memory', 'vm_swappiness',
+                    'vm.overcommit_ratio']
+SENZA_SYSCTL_CONF = '/etc/sysctl.d/99-senza.conf'
+
+
+def write_file(path, content):
+    with open(path, 'w') as file:
+        file.write(content)
+
+
+def main():
+    """If sysctl configuration is specified, writes sysctl configuration and reloads sysctl"""
+    configure_logging()
+    config = get_config()
+
+    sysctl = config.get('sysctl')
+
+    if sysctl is None:
+        sys.exit(0)
+
+    disallowed_keys = set(sysctl.keys()) - set(SYSCTL_WHITELIST)
+    if disallowed_keys:
+        logging.error('You are not allowed to configure the sysctl parameters {}'.format(list(disallowed_keys)))
+        sys.exit(1)
```

Comment:
I'm not sure exit(1) is a sane choice here. I'd rather configure the remaining parameters and show a warning instead of an error.